### DISCUSSION TITLE
Syphon / Spout & NDI capability for the Visual Synth

### DISCUSSION THREAD

**Discussion by producation** [spensbot/captivate#31] [Ideas]
To improve the Visual Synth I think it could be nice to stream textures into Captivate besides the local media. I would suggest to get
**Syphon** (_macOS_), **Spout** (_Windows_) and **NDI** (_macOS & Windows_) support for receiving a video signal into the Visual Scenes for further editing and the possibility to output the video stream (for e.g. projection mapping).
Syphon & Spout are processed by the GPU so it's very efficient.
NDI is processed by the CPU and adds some latency but it's widely supported and commonly used to stream cameras and every window.

**Comment by VisualBoy** [comment]
It would be cool to have NDI or Spout/Syphon  output for  routing video into others VJ software, like Resolume
